bot.py

```diff
@@ -1917,3 +1920,13 @@ async def on_ready():
     )
 
 bot.run(TOKEN)
+bot = commands.Bot(command_prefix="!")
```

Comment:
Discord has let developers know that prefix commands are no longer supported as of August 30th. For that reason, commands must be slash commands or context menu commands. For this command, it makes sense to make it a [Message Context Menu](https://docs.pycord.dev/en/master/api.html#messagecommand) command, which you can find a guide for [here](https://guide.pycord.dev/interactions/application-commands/context-menus#message-commands).